host/xtest/xtest_main.c

```diff
@@ -97,6 +102,12 @@ void usage(char *program)
 #endif
 	printf("\t--stats [opts]     Various statistics ('-h' for usage)\n");
 	printf("\n");
+	printf("Examples:\n");
+	printf("\txtest -t regression 4001 4003\n");
+	printf("\t                   run regression tests 4001 and 4003\n");
+	printf("\txtest -t regression -x 1017 -x 1018\n");
```

Comment:
s/1017/1027/
s/1018/1028/

Perhaps remove `-t regression` since it is the default? (and above also).

Reply:
for the example, we need `-t regression` to restrict added tests to the regression test suite only.